scripts/rmc-regression.sh

```diff
@@ -22,3 +22,6 @@ check-cbmc-viewer-version.py --major 2 --minor 5
 # Standalone rmc tests, expected tests, and cargo tests
 ./x.py build -i --stage 1 library/std ${EXTRA_X_PY_BUILD_ARGS}
 ./x.py test -i --stage 1 cbmc firecracker prusti smack expected cargo-rmc
+
+# Check codegen of firecracker
+$SCRIPT_DIR/codegen-firecracker.sh
```

Comment:
What about adding this as a separate workflow?

Reply:
Yes, can do. I was putting it into the regression to avoid rebuilding RMC in two different flows. Is there a way we can share artifacts across?

Reply:
We have been looking into this but we are not sure if it's good for caching RMC builds: https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts

But since it's not clear, any solution would be fine with me. I would like to revisit in the future to have 1 workflow that builds and multiple workflows which depend on that one to execute the regression.